postcss.config.mjs

```diff
@@ -2,7 +2,7 @@ import sass from '@csstools/postcss-sass';
 import autoprefixer from 'autoprefixer';
 import cssnano from 'cssnano';
 import color from 'postcss-color-rgb';
-import comment from 'postcss-comment';
```

Comment:
This package hasn't been updated in years, so I replaced it with `postcss-scss` which does handle inline comments as well.